codeflash/verification/test_runner.py

```diff
@@ -252,13 +256,13 @@ def run_benchmarking_tests(
         pytest_test_env = test_env.copy()
         pytest_test_env["PYTEST_PLUGINS"] = "codeflash.verification.pytest_plugin"
         blocklist_args = [f"-p no:{plugin}" for plugin in BENCHMARKING_BLOCKLISTED_PLUGINS]
-
-        results = execute_test_subprocess(
-            pytest_cmd_list + pytest_args + blocklist_args + result_args + test_files,
-            cwd=cwd,
-            env=pytest_test_env,
-            timeout=600,  # TODO: Make this dynamic
-        )
+        with custom_addopts():
```

Comment:
instead of doing this every time outside, move this custom_addopts inside the execute_test_subprocess function